pkg/apis/agones/v1/gameserver_test.go

```diff
@@ -358,17 +427,22 @@ func TestGameServerApplyDefaults(t *testing.T) {
 }
 
 func TestGameServerValidate(t *testing.T) {
-	gs := GameServer{
+	t.Parallel()
+	var fields []string
```

Comment:
Should this be turned into a table based test, since it's grown so large?

Reply:
I thought about it... but wasn't sure if it is worth spending extra time on that. But once you mentioned it - let's do that.